### PR TITLE
uk: shorter IAP descriptions

### PR DESCRIPTION
**Pull request by zemlanin**
to fit into 45 characters

### CODE REVIEW

File: CallsheetLocalizations/uk.xcloc/Localized Contents/uk.xliff [hunk 1]
```diff
@@ -716,7 +716,7 @@
       </trans-unit>
       <trans-unit id="IAP: Monthly Description" xml:space="preserve">
         <source>Unlimited searches for a month. Auto-renews</source>
-        <target state="translated">Пошук без обмежень на місяць. Автоматично подовжується</target>
+        <target state="translated">Рік без обмежень. Автоматично оновлюється</target>
```
Comment (zemlanin) [inline CallsheetLocalizations/uk.xcloc/Localized Contents/uk.xliff:719]: "A year without limits. Automatically renews"

File: CallsheetLocalizations/uk.xcloc/Localized Contents/uk.xliff [hunk 2]
```diff
@@ -716,7 +716,7 @@
       </trans-unit>
       <trans-unit id="IAP: Monthly Description" xml:space="preserve">
         <source>Unlimited searches for a month. Auto-renews</source>
-        <target state="translated">Пошук без обмежень на місяць. Автоматично подовжується</target>
+        <target state="translated">Місяць без обмежень. Автоматично оновлюється</target>
```
Comment (zemlanin) [inline CallsheetLocalizations/uk.xcloc/Localized Contents/uk.xliff:719]: "A month without limits. Automatically renews"

File: CallsheetLocalizations/uk.xcloc/Localized Contents/uk.xliff [hunk 3]
```diff
@@ -726,7 +726,7 @@
       </trans-unit>
       <trans-unit id="IAP: Yearly Description" xml:space="preserve">
         <source>Access to Callsheet for 1 year. Auto-renews</source>
-        <target state="translated">Доступ до Callsheet на рік. Автоматично подовжується</target>
+        <target state="translated">Рік без обмежень. Автоматично оновлюється</target>
```
Comment (zemlanin) [inline CallsheetLocalizations/uk.xcloc/Localized Contents/uk.xliff:729]: "A year without limits. Automatically renews"

**Comment by cliss** [comment]
Excellent; thanks! FWIW, I've added to the comment — but you haven't seen it yet — that these should be 45 or less.

**Pull request merged**